auto/commands/TorqueHolonomicDriveController.java

```diff
@@ -0,0 +1,95 @@
+package org.texastorque.torquelib.auto.commands;
+
+import java.sql.Driver;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.path.PathPlannerPath;
+
+// Class adapted from PPHolonomicDriveController from PathplannerLib
+
+import com.pathplanner.lib.path.PathPlannerTrajectory;
+import com.pathplanner.lib.path.PathPlannerTrajectory.State;
+import com.pathplanner.lib.path.PathPoint;
+import com.pathplanner.lib.util.PIDConstants;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
```

Comment:
Do you actually understand whats going on here?

Reply:
I believe so, at least enough to get it working on bucket bot. I'll read through it more 👍